update docs/_layouts/page.html : removed redundantlinks

shubham-tyagi opened the pull request:
removed redundant links from TOC of every docs page.

Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**

This PR fixes #4314 

**Notes for Reviewers**


**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit 5fdc0717fbb58d96f2545c186d2f9da02cf3a8af at: https://6162cdc5243e77e71e3d9844--meshery-docs-preview.netlify.app

shubham-tyagi commented:
Hey @leecalcote , I have removed TOC from "page" layout which uses "default" as its layout. In the "default" layout TOC is already included but we were including TOC again on the "page" itself. So I removed the TOC included in "page". I don't know if I got what you are saying right, but if a doc page is not using "page" layout how will that affect any other page because I made changes in "page" layout. Please correct me if I answered a question you didn't ask.
